about.html

```diff
@@ -0,0 +1,163 @@
+            <li>
+              <a href="#">join</a>
+            </li>
+            <li>
+              <a href="#">sponser</a>
+            </li>
+            <li>
+              <a href="#">news</a>
+            </li>
+            <li class="cc">
+              <img src="images/CC Campaign@2x.png" alt="">
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <!--end of desktop menu-->
+    </header>
+    <main class="container">
+      <section class="banner">
+        <div class="intro margintop">
+          <p class="center">&quot;Hello! Great people&quot;</p>
+          <h1 class="center">Celebrating Excellence:<br> Our Prize Giving Ceremony</h1>
+        </div>
+        <div class="discription center desktopDescription">
+          <p>Discover extraordinary achievements at our Prize Giving Ceremony, where we applaud the best and brightest. Join us in
+          celebrating excellence, as we inspire a community of driven individuals to reach new heights. Embrace a night of
+          recognition, inspiration, and triumph in the company of exceptional talent.
+          <div class=""></div>
+          </p>
+        </div>
+      </section>
+      <div class="email_contact">
+        <p>Please contact us per Email for nay further questions about CC Global Summit 2022!</p>
+        <div class="contactEmail">
+          <p class="center">ccsummit2015@cckorea.org</p>
+        </div>
+      </div>
+      <section class="summits_info">
+        <article class="logoInfo">
+          <h3>Creative Commons <br>
+            Global Summit 2015 Logo</h3>
+          <div class="divLine"></div>
+          <p>The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+          <div class="summit_logo">
+            <img src="images/about_logo 1.png" alt="summits logo 2015">
+          </div>
+        </article>
+        <article class="previpusSummit">
+          <h3>See the past CC Global Summits</h3>
+          <p>Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+          <div class="prevSummit Images">
+            <img src="images/about_2011 1.png" alt="sumits image 2011">
+            <img src="images/about_2013 1.png" alt="summits image 2013">
+          </div>
+        </article>
+        <article class="partners aboutpartners">
+          <h5>Patner</h5>
+          <div class="divLine"></div>
+          <div class="patnerslogo">
+            <img src="images/mozilla 1.png" alt=" mozila logo">
+            <img src="images/airbnb 1.png" alt="airbnb logo">
+            <img src="images/googlekorea 1.png" alt="googlekorea logo">
+            <img src="images/daumkakaologo 1.png" alt="daumkakaologo">
+            <img src="images/naver 1.png" alt="naver logo">
+          </div>
+        </article>
+      </section>
+    </main>
+    <footer class="aboutfooter">
+      <div class="footer">
+        <a href="index.html">TriumphTribute</a>
```

Comment:
[ ] Kindly change the footer's color on the About page ( desktop version) to black to match the design provided.🤓
your design 👇:
![his](https://github.com/MPDADDY/capstone_prize-giving_day/assets/62156607/29b56a4e-e4f4-47e8-9acc-90d97d99a51b)

Figma design 👇 :
![design](https://github.com/MPDADDY/capstone_prize-giving_day/assets/62156607/5a43826b-1167-4815-8cdd-cfc96adde2f8)